external/loaders/loaders/batches/_batch.py

```diff
@@ -74,11 +75,10 @@ def batches_from_geodata(
     data_mapping = _create_mapper(data_path, mapping_function, mapping_kwargs)
     batches = batches_from_mapper(
         data_mapping,
-        variable_names,
-        timesteps_per_batch,
-        random_seed,
-        timesteps,
-        res,
+        variable_names=variable_names,
+        timesteps_per_batch=timesteps_per_batch,
+        timesteps=timesteps,
+        res=res,
```

Comment:
There are some cases (surface flux training data) where people use `batches_from_geodata` with `open_zarr` as the mapping function. Can you also pass the new `batches_from_mapper` args for `unstacked_dims, subsample_ratio, drop_nans` here and add them as optional args to `batches_from_geodata`?

Not relevant to this PR, but I was reminded of this issue I created noting that `batches_from_geodata` is somewhat redundant now that `batches_from_mapper` can return unstacked data, and could be removed (https://github.com/ai2cm/fv3net/issues/1430).

Reply:
Updated. As you noted, it should be possible to write any config that uses `batches_from_geodata` to use `batches_from_mapper` instead. If it is, I would suggest doing that instead of adding these arguments to a config using `batches_from_geodata`.